### PR TITLE
feat(discover2) Change field aliases to be parameterless functions

### PR DESCRIPTION
**Pull request by evanh**
Change the field aliases (p95 etc) to be parameterless functions to better show
to the user that they are not simple fields.

This is purely a backend change right now (plus updating the default graphs).

### CODE REVIEW

File: src/sentry/static/sentry/app/views/eventsV2/data.tsx [hunk 1]
```diff
@@ -11,12 +11,12 @@ import UserBadge from 'app/components/idBadge/userBadge';
 import getDynamicText from 'app/utils/getDynamicText';
 import overflowEllipsis from 'app/styles/overflowEllipsis';
 import pinIcon from 'app/../images/graph/icon-location-filled.svg';
-import {Organization, NewQuery} from 'app/types';
+import { Organization, NewQuery } from 'app/types';
```
Comment (markstory) [inline src/sentry/static/sentry/app/views/eventsV2/data.tsx]: You might want to check that your editor is using the same prettier version/configuration as our application does. These whitespace changes will likely enrage prettier.

File: src/sentry/api/event_search.py [hunk 2]
```diff
@@ -893,21 +910,23 @@ def resolve_field(field):
     if not isinstance(field, six.string_types):
         raise InvalidSearchQuery("Field names must be strings")
 
-    if field in FIELD_ALIASES:
-        special_field = deepcopy(FIELD_ALIASES[field])
-        return (special_field.get("fields", []), special_field.get("aggregations", []))
+    sans_parens = field.strip("()")
+    if sans_parens in FIELD_ALIASES:
```
Comment (markstory) [inline src/sentry/api/event_search.py]: Would having both `p95` and `p95()` in the FIELD_ALIASES map make this logic simpler?

File: src/sentry/static/sentry/app/views/eventsV2/data.tsx (hunk 1 above)
Comment (evanh) [reply src/sentry/static/sentry/app/views/eventsV2/data.tsx]: Yeah noted. I'll upgrade.

File: src/sentry/api/event_search.py (hunk 2 above)
Comment (evanh) [reply src/sentry/api/event_search.py]: No I don't think so. This check is largely here to track whether or not we removed brackets before sending to snuba, so we can add them back when returning the results to the frontend.

Comment (evanh) [reply src/sentry/api/event_search.py]: Turns out I didn't even need the check, the frontend strips the brackets anyways.

File: tests/sentry/snuba/test_discover.py [hunk 3]
```diff
@@ -349,6 +349,39 @@ def test_selected_columns_aggregate_alias(self, mock_query):
             referrer=None,
         )
 
+    @patch("sentry.snuba.discover.raw_query")
+    def test_selected_columns_aggregate_alias_with_brackets(self, mock_query):
+        mock_query.return_value = {
+            "meta": [{"name": "transaction"}, {"name": "p95()"}],
+            "data": [{"transaction": "api.do_things", "p95()": 200}],
```
Comment (markstory) [inline tests/sentry/snuba/test_discover.py]: Should this be `p95` without parens?

**Review by markstory — APPROVED** [review]

File: tests/sentry/snuba/test_discover.py (hunk 3 above)
Comment (evanh) [reply tests/sentry/snuba/test_discover.py]: If I were concerned about exactly matching what the backend returns then right now yes. Of course we don't use this value so it doesn't matter right now.

**Pull request merged**